README.md

```diff
@@ -362,16 +366,35 @@ unzip bowtie2.zip
 rm bowtie2.zip
 ```
 
-#### Set up Bowtie2 database
+#### Download pre-built human genome database
 
-To create a Bowtie2 database, we need to download genomes from NCBI, using ncbi-genome-download. To run the affiliated script `gimme_taxa.py`, you will also need to install the dependencies `ete3` and `six`. 
+For detecting human reads we use the standard pre-built "Human / GRCh38 no-alt
```

Comment:
This is a detail, but at the Nextflow conference there was a talk by Adam Phillippy who is the lead investigator behind the fully sequenced human genome, published in 2022. https://www.science.org/doi/10.1126/science.abj6987

This genome covers bits of the genome that were previously not covered:
"The completed regions include all centromeric satellite arrays, recent segmental duplications, and the short arms of all five acrocentric chromosomes, unlocking these complex regions of the genome to variational and functional studies."

In case you want to use it, it's also offered on Index zone: https://benlangmead.github.io/aws-indexes/bowtie#:~:text=rev.2.bt2-,Human%20/%20CHM13plusY,-T2T3

Reply:
Thanks for looking into this; switched!